lib/ansible/modules/cloud/azure/azure_rm_subnet_facts.py

```diff
@@ -189,8 +189,8 @@ def get(self):
                                                        virtual_network_name=self.virtual_network_name,
                                                        subnet_name=self.name)
             self.log("Response : {0}".format(response))
-        except CloudError as e:
-            self.fail('Could not get facts for Subnet.')
+        except CloudError:
```

Comment:
just wondering about one scenario.
what if get() fails because of network error?
what kind of errors are possible here?
I am mostly concerned about errors other than 404, when resource exists, but query failed for some other reason.